server/service/store/handler_store_data.go

```diff
@@ -0,0 +1,62 @@
+package store
+
+import (
+	"encoding/hex"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/gorilla/mux"
+	log "github.com/sirupsen/logrus"
+)
+
+func HandleStoreData(w http.ResponseWriter, r *http.Request) {
+	dataHashStr := mux.Vars(r)["key"]
```

Comment:
I'm confused as to whether this name is a `key` or a `dataHash`.
Can you unify your variable names?

Reply:
I think `dataHash` seems to be more better. I'll change it!